PokeApiNet/Models/Pokemon.cs

```diff
@@ -146,6 +146,17 @@ public class Characteristic : ApiResource
         /// </summary>
         [JsonProperty("possible_values")]
         public List<int> PossibleValues { get; set; }
+
+        /// <summary>
+        /// The highest stat of this characteristic.
+        /// </summary>
+        [JsonProperty("highest_stat")]
+        public Stat HighestStat { get; set; }
```

Comment:
`highest_stat` only contains `name` and `url`, making this property type `NamedApiResource<Stat>`. See [here](https://github.com/PokeAPI/pokeapi/blob/a5205b512640bd33684acbbb3f23a8d98aee0cbc/pokemon_v2/serializers.py#L323) and [here](https://github.com/PokeAPI/pokeapi/blob/a5205b512640bd33684acbbb3f23a8d98aee0cbc/pokemon_v2/serializers.py#L488)

```suggestion
        public NamedApiResource<Stat> HighestStat { get; set; }
```